frontend/packages/operator-lifecycle-manager/src/types.ts

```diff
@@ -294,6 +294,6 @@ export type OperatorGroupKind = {
 
 export type DeprecatedOperatorWarning = {
   deprecation?: {
-    message: string;
+    message?: string;
```

Comment:
Changed to optional since "Property 'message' is optional in type 'K8sResourceCondition' but required in type '{ deprecation.message: string; }'" API